.changes/v0.12.0.md

```diff
@@ -0,0 +1,74 @@
+## v0.12.0 - 2024-07-12
+
+### 🔥 Breaking Changes
+
+- sdks: Various breaking changes to SDKs \
+  See SDK-specific release notes for more information
```

Comment:
The breaking changes in Python and TS are just about the return `Void` thing which isn't really breaking in these SDKs. In Go, in addition to that, it's the aliases removal thing but that should only affect Modules. And even on the `Void` return, it only affects these core APIs related to modules:
- `Module.serve` (very advanced use case)
- `Service.up` (for CLI)
- `FunctionCall.returnValue` (for SDKs)

Doesn't seem worth mentioning other than Go for those two things. I guess I’m being mostly picky with "Various" because it seems to be more than it is.

Reply:
Scoped this down, and just mentioned go.